cli/tests/integrations/test_marathon_pod.py

```diff
@@ -64,31 +64,23 @@ def test_pod_show():
         _assert_pod_show(GOOD_POD_ID, expected_json)
 
 
-@pytest.mark.skipif(not _PODS_ENABLED, reason="Requires pods")
-def test_pod_update_from_properties():
-    expected_json = file_json_ast(UPDATED_GOOD_POD_FILE_PATH)
-    containers_json_str = json.dumps(expected_json['containers'])
-    networks_json_str = json.dumps(expected_json['networks'])
-    properties = ['id=/{}'.format(GOOD_POD_ID),
-                  'containers={}'.format(containers_json_str),
-                  'networks={}'.format(networks_json_str)]
+def test_pod_update_does_not_support_properties():
```

Comment:
I was thinking it would be good to keep around to prevent people from adding properties back to `pod update` in the future. I could see someone doing that in an effort to keep all the `update` commands consistent.